Gordon360/Utilities/ImageUtils.cs

```diff
@@ -20,7 +21,11 @@ public static void DeleteImage(string imagePath)
         {
             try
             {
-                File.Delete(imagePath);
+                // remove server address from imagePath, otherwise Student News image not found
+                string splitSubstring = "browseable";
+                string realImagePath = splitSubstring + imagePath.Split(splitSubstring).Last();
+
+                File.Delete(realImagePath);
```

Comment:
This file probably shouldn't be changed just to fix a particular issue with how you are calling it in student news. I would rather have you change the URL to a local file path in student news so that you are always passing a file path to this function.